org.zotero.Zotero.json

```diff
@@ -0,0 +1,41 @@
+{
+    "id": "org.zotero.Zotero",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "zotero",
+    "rename-desktop-file": "zotero.desktop",
+    "rename-icon": "zotero",
+    "finish-args": [
+        "--socket=x11",
+        "--share=ipc",
+        "--share=network",
+        "--filesystem=home"
+    ],
+    "modules": [
+        {
+            "name": "zotero",
+            "buildsystem": "simple",
+            "sources": [{
+                "type": "archive",
+                "url": "https://download.zotero.org/client/release/5.0.34/Zotero-5.0.34_linux-x86_64.tar.bz2",
+                "sha256": "df0c5aa38b7e2ec2a41006f7cb8616fce3956f452c19125dc89c66b11c51e392"
+            },
+            {
+                "type": "file",
+                "path": "org.zotero.Zotero.appdata.xml"
+            }],
+            "build-commands": [
+                "mkdir -p /app/bin /app/share /app/share/icons/hicolor/{16x16,32x32,48x48,256x256}/apps /app/share/appdata",
+                "cp -R . /app/share/zotero",
+                "cp chrome/icons/default/default16.png /app/share/icons/hicolor/16x16/apps/zotero.png",
```

Comment:
I prefer using `install` rather than `cp` when possible as it can do things like creating the directory in a single call instead of `mkdir` as above.

Reply:
I absolutely agree here.